Makefile

```diff
@@ -71,65 +73,76 @@ HELM_VER = v3.5.3
 TIMEOUT = 15
 Q = $(if $(filter 1,$V),,@)
 
-## Options for 'bundle-build'
-#ifneq ($(origin CHANNELS), undefined)
-#BUNDLE_CHANNELS := --channels=$(CHANNELS)
-#endif
-#ifneq ($(origin DEFAULT_CHANNEL), undefined)
-#BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
-#endif
-#BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+# Docker vars
+IMAGE_BUILDER ?= podman
```

Comment:
keep default as before (docker) ? we can change it later if needed but it will require changes in our CI

Reply:
Changed to `docker`